extensions/vscode/src/language-server/index.ts

```diff
@@ -0,0 +1,54 @@
+import { createConnection, createServer, createSimpleProject } from '@volar/language-server/node';
+import { create as createYamlService } from 'volar-service-yaml';
+import { SchemaPriority } from 'yaml-language-server';
+import { frontmatterPlugin } from './languagePlugin';
+import { readSchema } from './schema';
+
+const connection = createConnection();
+const server = createServer(connection);
+
+connection.listen();
+
+connection.onInitialize((params) => {
+  const yamlService = createYamlService({
+    getLanguageSettings(_context) {
+      const schema = readSchema();
+
+      return {
+        completion: true,
+        validate: true,
+        hover: true,
+        format: true,
+        yamlVersion: '1.2',
+        isKubernetes: false,
+        schemas: [
+          {
+            uri: 'https://tutorialkit.dev/reference/configuration/',
+            schema,
+            fileMatch: [
+              '**/*',
+
+              // TODO: those don't work
+              'src/content/*.md',
+              'src/content/**/*.md',
+              'src/content/**/*.mdx',
```

Comment:
I still need to investigate that, ideally I don't want the check to apply to every markdown & mdx files.